dhctl/pkg/preflight/ssh.go

```diff
@@ -64,7 +68,24 @@ func (pc *Checker) CheckSSHCredential() error {
 	sshCheck := pc.sshClient.Check()
 	err := sshCheck.CheckAvailability()
 	if err != nil {
-		return fmt.Errorf("ssh %w. Please check ssh credential and try again", ErrAuthFailed)
+		return fmt.Errorf(
+			"ssh %w. Please check ssh credential and try again",
+			ErrAuthSSHFailed,
+		)
+	}
+	return nil
+}
+
+func (pc *Checker) CheckSingleSSHHostForStatic() error {
+	if app.PreflightSkipOneSSHHost {
+		log.InfoLn("Only one --ssh-host parameter used preflight check was skipped")
+		return nil
+	}
+
+	if len(pc.sshClient.Settings.AvailableHosts()) > 1 {
+		return fmt.Errorf(
+			"for the bootstrap first master, only one --ssh-host parameter should be used for a static cluster",
```

Comment:
"During the bootstrap of the first static master node, only one --ssh-host parameter is allowed"